metadata/political/senate_districts_2022_to_2032.md

```diff
@@ -0,0 +1,73 @@
+# Title
+
+Utah Senate Districts
+
+## ID
+
+1d166196-da46-40e0-9c4c-8f2524d39aff
+
+## Brief Summary
+
+Utah Senate Districts defined after every decennial census.
+
+## Summary
+
+This dataset contains Utah State Senate district boundaries for the years 2022-2032. Statewide Senate district boundaries are drawn by the [Utah State Legislature](https://le.utah.gov/) after each decennial census and adopted into State law.
+
+## Description
+
+### What is the dataset?
+
+The [Utah State Senate](https://senate.utah.gov/) is comprised of 29 men and women elected to four-year terms. This dataset shows the geographic boundaries of Senate districts that will be used from 2022 to 2032.
+
+### What is the purpose of the dataset?
+
+The data will be used for election purposes beginning January 1, 2022. Elected officials began representing these districts in January 2023. These boundaries supersede the State Senate districts that were used in 2012-2021. This dataset also helps citizens [find their senators](https://le.utah.gov/GIS/findDistrict.jsp) and is suitable for general analytic purposes.
+
+### What does the dataset represent?
+
+Each polygon in this dataset represents the approximate geographic boundary that a given senator represents. The DIST field contains the House district number. UGRC also included the COLOR4 field so that you can recolor the polygons so that no two adjacent districts display with the same color.
```

Comment:
```suggestion
Each polygon in this dataset represents the approximate geographic boundary that a given senator represents. The DIST field contains the House district number. UGRC also included the COLOR4 field so that you can symbolize the polygons so that no two adjacent districts display with the same color.
```